bigbluebutton-html5/imports/ui/components/app/component.jsx

```diff
@@ -224,7 +224,7 @@ class App extends Component {
 
     // Variables for resizing chat.
     const CHAT_MIN_WIDTH_PX = 180;
-    const CHAT_MAX_WIDTH_PX = 310;
+    const CHAT_MAX_WIDTH_PX = 600;
```

Comment:
Does the number have to be pixels? I had problems on a 1440 monitor and a 4k monitor would probably continue to have issues.

Reply:
For reference this is what I am seeing for 4k.

![4k](https://user-images.githubusercontent.com/32152296/44927072-45b7ce80-ad21-11e8-9e11-9fa80d0e11d6.gif)

Reply:
The issue is related to `vw` and `vh` units, you can pass the percentage as string to the Re-sizable puglin. 
```
Specifies the width and height that the dragged item should start at. For example, you can set 300, '300px', 50%. If both defaultSize and size omitted, set 'auto'.

```
[ref](https://github.com/bokuweb/re-resizable#defaultsize--width-number--string-height-number--string-)